index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title> My Portfolio </title>
+</head>
+<body>
+    <header id="header">
+        <div class="head">
+            <h1 id="logo">Fredo</h1>
+            <img class="menu-icon" src="iconMenu.svg" alt="menuIcon">
+        </div>
+        <section class="presentation">
+            <div class="greeting">
+                I’m Fredo Glad to see you!
+            </div>
+            <div class="self-description">
+                I’m a full stack software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
+            <div class="media-connect">
+                <p>LET’S CONNECT</p>
+                <nav class="media-list">
+                    <a href="https://twitter.com/Fredo_st_fleur?t=OEPwMJjspBTEnTevArv0HA&s=09" rel=noopener target="_blank" class="media-link"> <img src="twitterIcon.svg" alt="twitter-icon"> </a>
+                    <a href="https://www.linkedin.com/in/fredo-st-fleur-0b41a122a" rel=noopener target="_blank" class="media-link"> <img src="linkedIncon.svg" alt="linkdln-icon"> </a>
+                    <a href="mailto:fredostfleur23@gmail.com" rel=noopener target="_blank" class="media-link"> <img src="mailIcon.svg" alt="mail-icon"> </a>
+                    <a href="https://github.com/fredo509" rel=noopener target="_blank" class="media-link"> <img src="githubIcon.svg" alt="mail-icon"> </a>
+                </nav>
```

Comment:
- [ ] When listing items on a webpage, it is semantic to use `ul` and `li` tags or `ol` and `li` tags if you want the items ordered. You don't have to remove the `a` tags and the `img` tags, all you have to do is wrap them with the `ul` and `li` tags. So the structure would be like this:

```HTML
<ul>
  <li>
    <a href="mailto:fredostfleur23@gmail.com" rel=noopener target="_blank" class="media-link"> <img src="mailIcon.svg" alt="mail-icon"> </a>
  </li>
  <li>
    <a href="https://github.com/fredo509" rel=noopener target="_blank" class="media-link"> <img src="githubIcon.svg" alt="mail-icon"> </a>
  </li>
  ...
</ul>
```